src/index.ts

```diff
@@ -26,6 +27,16 @@ function decodeCerthashStr (s: string): MultihashDigest {
   return digest.decode(multibaseDecoder.decode(s))
 }
 
+// The same as 'err-code' package
+function errCode (err: Error, code: string): Error {
+  Object.defineProperty(err, 'code', {
```

Comment:
Please avoid `Object.defineProperty` - https://humanwhocodes.com/blog/2015/11/performance-implication-object-defineproperty/

Reply:
Thanks! I copied what err-code package did: https://github.com/IndigoUnited/js-err-code/blob/master/index.js#L17 but we can just set `err.code = ` it's okay if this is enumerable. 

But also this is only happens when we return an error, so shouldn't affect perf at all, right?

Reply:
I removed this since I don't think it's useful. I was following the pattern from mplex, but it's not necessary.